src/api/plugins.js

```diff
@@ -7,6 +7,14 @@ const { PLUGIN_PATH } = require('../utils/paths');
 const config = new Conf();
 
 const getAll = () => config.get('plugins') || [];
+const getAllPlugins = () => {
```

Comment:
Let's rename this to `fetchPlugins()` instead as to not create any confusion between `getAll` and `getAllPlugins`